runtime/src/bank.rs

```diff
@@ -2144,6 +2179,7 @@ impl From<LegacyBank0223> for Bank {
             storage_accounts: bank.storage_accounts,
             parent_hash: bank.parent_hash,
             parent_slot: bank.parent_slot,
+            hard_forks: Arc::new(RwLock::new(HardForksMap::new())),
```

Comment:
keeping good old LegacyBank0223 compatible, the nice thing is that rustc yelled at me to include this new field